scripts/createNewComponent.js

```diff
@@ -35,10 +35,11 @@ const createComponentFile = () => {
   const content = `
     import { LitElement } from 'lit';
     import { customElement } from 'lit/decorators.js';
+    import { INveComponent } from '@interfaces/NveComponent.interface';
     import styles from './${componentName}.styles';
 
     @customElement('${componentName}')
-    export default class ${className} extends LitElement {
+    export default class ${className} extends LitElement implements INveComponent {
```

Comment:
Her kan du gjøre det enda enklere for utvikleren ved å også implementere testId:
![image](https://github.com/NVE/Designsystem/assets/71138449/6454d254-dafd-4a7b-b9eb-1769ffc159ff)

Reply:
Enig, lægger til det.